tests/translator/input/graphqlapi_resolver_function_with_lambda_datasource.yaml

```diff
@@ -26,9 +26,20 @@ Resources:
           myResolver:
             FieldName: my_field
             MaxBatchSize: 10
+            Caching:
+              Ttl: 20
+              CachingKeys:
+              - key1
+              - key2
+            Pipeline:
+            - MyFunction
+          myResolverWithCustomCode:
+            FieldName: my_field
+            InlineCode: custom code
             Runtime:
-              Name: some-runtime
+              Name: APPSYNC_JS
```

Comment:
nit: I'd prefer adding entirely new transform tests instead of modifying old ones, otherwise it's harder to verify at a glance that it contains no backward incompatible changes

Reply:
Thanks. I agree. I'll keep it in mind for future changes. As for this change, there is no incompatible changes. The change above is because using "some-runtime" is actually incorrect value, a random text